.github/workflows/docs.yaml

```
@@ -0,0 +1,49 @@
+name: Docs
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  docs:
+    runs-on: ubuntu-22.04
+    name: Build and validate documentation
```

The name typically comes first for readability:
```suggestion
    name: Build and validate documentation
    runs-on: ubuntu-22.04
    
```